### ISSUE TITLE
Doesn't work with rollup-plugin-node-resolve

### ISSUE THREAD

**Issue by nicolasparada** [rollup/rollup-watch#3]
If I use it alongside with `rollup-plugin-node-resolve`, it doesn't work. In the console I got printed `bundling...` and just finished there, without any error message.

I was using a couple of plugins and just disabled each one until I found that `node-resolver` is the responsible.

``` js
"rollup": "^0.31.0",
"rollup-plugin-node-resolve": "^1.7.0",
"rollup-watch": "^2.1.1"
```

**Comment by nicolasparada** [comment]
Now with `rollup-watch` version 2.2.0 I get the error: 

``` js
unknown event { code: 'ERROR',
  error:
   { Error: Path must be a string without null bytes
       at ...
       ... code: 'ENOENT' } }
```

**Comment by Rich-Harris** [comment]
I'm struggling to reproduce this – https://github.com/Rich-Harris/rollup-watch-gh-3-repro – can you help? thanks

**Comment by nicolasparada** [comment]
Yes, just added a real npm depencency like `vuex-router-sync` and it fails with the before mentioned error.
But, without the `-w` flag, it compiles.

``` js
import { sync } from 'vuex-router-sync'
sync(undefined, undefined)
```

``` bash
unknown event { code: 'ERROR',
  error:
   { Error: Path must be a string without null bytes
       at nullCheck (fs.js:129:14)
       at Object.fs.watch (fs.js:1434:3)
       at new FileWatcher (C:\Users\Nicolas\Dev\rollup-watch-gh-3-repro\node_modules\rollup-watch\dist\rollup-watch.cjs.js:47:5)
       at C:\Users\Nicolas\Dev\rollup-watch-gh-3-repro\node_modules\rollup-watch\dist\rollup-watch.cjs.js:106:23
       at Array.forEach (native)
       at C:\Users\Nicolas\Dev\rollup-watch-gh-3-repro\node_modules\rollup-watch\dist\rollup-watch.cjs.js:102:22 code: 'ENOENT' } }
```

**Comment by jacobmischka** [comment]
```js
import { sync } from 'vuex-router-sync';

sync(undefined, undefined);
```

```
Module /home/mischka/projects/rollup-watch-gh-3-repro/node_modules/vuex-router-sync/index.js does not export sync (imported by /home/mischka/projects/rollup-watch-gh-3-repro/src/main.js)
```

**Comment by jacobmischka** [comment]
Edit: Ah, nevermind, I didn't realize the config didn't already include the commonjs plugin. Getting the same error as well now on Linux.

**Comment by jacobmischka** [comment]
The problem seems to be with commonjsHelpers, for some reason its `file` that is being passed to the FileWatcher is '\u0000commonjsHelpers', which is breaking fs.watch. 

So the problem is the commonjs plugin, not the node resolve plugin:
https://github.com/rollup/rollup-plugin-commonjs/blob/master/src/index.js#L48

**Comment by Rich-Harris** [comment]
Ah, that makes sense – thanks. Released 2.4.0 which should fix this, as it disregards paths with `\u0000` (convention to signify plugin helper modules) and ignores IDs that aren't file paths. Reopen if not!
